src/Playwright.NUnit/PlaywrightTestAttribute.cs

```diff
@@ -76,6 +83,13 @@ public override TestResult Execute(TestExecutionContext context)
                 {
                     break;
                 }
+                // NUnit takes care about running TearDown for the last iteration. PlaywrightTestAttribute inherits from
+                // the TimeoutCommand which runs TearDown for us in a case of a timeout.
+                if (!TearDownRan)
```

Comment:
I think merging `RetryTestCommand` and `TimeoutCommand` could lead to easier code, wdyt?

Reply:
I made an attempt, seems better already.